### DISCUSSION TITLE
Confusion around the new TTS feature

### DISCUSSION THREAD

**Discussion by BritishTeapot** [LostRuins/koboldcpp#941] [Q&A] [answered]
Hi! Have been trying to make the new whisper.cpp powered TTS to work, but It just refuses to show up anywhere in the UI. When I click TTS option I only see options for XTTS and AllTalk servers, despite the koboldcpp reporting successful loading of the whisper model. What am I missing?

**Comment by BritishTeapot** [comment, marked as answer]
Okay just realized whisper is not TTS, but STT. Don't know how I could miss that :D

**Reply by LostRuins** [reply]
Yeah, TTS is provided by XTTS, alltalk or browser text to speech

Whisper (voice control) is STT. It supports push to talk and voice detection

**Reply by zcattacz** [reply]
But where is this button/toggle to push? I can't find it anywhere...
